sql/install.sql

```diff
@@ -4,4 +4,31 @@ CREATE TABLE users (
   email TEXT,
   password VARCHAR(32),
   PRIMARY KEY (user_id)
-)
\ No newline at end of file
+)
```

Comment:
make sure you add ";" to close previous query. I guess this was missed on that task, but it will fail if you want to run all queries from this file.